core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -121,7 +121,7 @@ protected[controller] object RestApiCommons {
  */
 protected[controller] trait RespondWithHeaders extends Directives {
     val allowOrigin = `Access-Control-Allow-Origin`(AllOrigins)
-    val allowHeaders = `Access-Control-Allow-Headers`("Authorization", "Content-Type")
+    val allowHeaders = `Access-Control-Allow-Headers`("Authorization", "Content-Type", "User-Agent")
```

Comment:
the controller does not log user agents - i don't think it should/needs to.
you get that from nginx.